src/pyatsimagebuilder/Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV TINI_VERSION=0.18.0
 ENV WORKSPACE="{workspace}"
 
 RUN apt-get update \
- && apt-get install -y --no-install-recommends iputils-ping telnet openssh-client curl build-essential \
+ && apt-get install -y --no-install-recommends iputils-ping telnet openssh-client curl build-essential net-tools \
```

Comment:
I think we can actually remove this file entirely. It was probably mistakenly not removed when the Dockerfile.temple file was created instead.

Reply:
delete the file..